README.md

```diff
@@ -46,6 +46,9 @@ var job = await client.submitJobLocalFile("./path/to/file.mp4");
 
 // or submit via a public url
 var job = await client.submitJobUrl("https://www.rev.ai/FTC_Sample_1.mp3");
+
+// or from audio data, this requires that you also provide a filename
+var job = await client.submitJobAudioData("Insert Buffer or ReadableStream", "filename.mp3");
```

Comment:
What do you think of having the example open a local file and pass the buffer through here instead?